### PR TITLE
Bug 1561694 - Switching hardcoded layout to `3-col-7-row-octr`

### PR DESCRIPTION
**Pull request by gvn**
Testing:

- Open FF and confirm that default layout matches [winning layout](https://www.figma.com/file/o1Ep3tU1pCsCgM0gWNe7Iu/2-Branch-Study-Firefox-v67)

### CODE REVIEW

**Comment by gvn** [comment]
@Mardak is this basically all that needs to be done for [this ticket](https://bugzilla.mozilla.org/show_bug.cgi?id=1551273)?

File: lib/DiscoveryStreamFeed.jsm [hunk 1]
```diff
@@ -1072,10 +1072,11 @@ this.DiscoveryStreamFeed = class DiscoveryStreamFeed {
   }
 };
 
+// Hardcoded version of layout_variant `3-col-7-row-octr`
 defaultLayoutResp = {
   "spocs": {
-    "url": "https://getpocket.cdn.mozilla.net/v3/firefox/unique-spocs",
-    "spocs_per_domain": 1,
+    "url": "https://getpocket.cdn.mozilla.net/v3/firefox/unique-spocs?consumer_key=$apiKey",
```
Comment (Mardak) [inline lib/DiscoveryStreamFeed.jsm]: Did this API change on the server to reject `consumer_key`-less requests? Any users already seeing discovery stream will be getting the `400 bad request` response -- expected?

File: lib/DiscoveryStreamFeed.jsm [hunk 2]
```diff
@@ -1100,204 +1101,92 @@ defaultLayoutResp = {
...
-    },
+            ".ds-navigation": "margin-top: -10px;"
+          }
+        }
+      ]
+    }
   ],
+  "feeds": {},
+  "error": 0,
+  "status": 1
```
Comment (Mardak) [inline lib/DiscoveryStreamFeed.jsm]: Are these fields intentionally added now? I believe @ScottDowne explicitly removed them before

Comment (gvn) [reply lib/DiscoveryStreamFeed.jsm]: This is just a copy of the layout JSON from the endpoint, so if it works without them I can remove them!

**Comment by gvn** [comment]
> The bug was filed to turn on discovery stream for all channels, but this PR seems to only change the layout. Are we not expecting to turn on the feature via normandy or some other way?

Ok, yeah this is what was confusing me. I changed the baked layout for this ticket based on our conversation in stand, but that didn't seem to be what the ticket was for exactly. Should this actually be 2 separate tickets? I'm unfamiliar with Normandy.

File: lib/DiscoveryStreamFeed.jsm (hunk 1 above)
Comment (Mardak) [reply lib/DiscoveryStreamFeed.jsm]: Actually, looks like spocs are currently requested via consumer_key as part of POST body. What is the server actually wanting? Both consumer_key in the url params and the post body?

Comment (gvn) [reply lib/DiscoveryStreamFeed.jsm]: @mmiermans Can you answer this Q?

File: lib/DiscoveryStreamFeed.jsm (hunk 2 above)
Comment (gvn) [reply lib/DiscoveryStreamFeed.jsm]: Works without them...removing.

File: lib/DiscoveryStreamFeed.jsm (hunk 1 above)
Comment (mmiermans) [reply lib/DiscoveryStreamFeed.jsm]: It hasn't changed; `consumer_key` has always been required. Without it, the response will be `400 Bad Request` and the `X-Error` header will say `Missing API key. Get an API key at http:\/\/getpocket.com\/api\/.`.

Good thing to call our in review, and thanks for fixing it @gvn!

**Pull request merged**